src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -162,13 +163,13 @@ export default class AirportModel {
         this._runwayCollection = null;
 
         /**
-         * Dictionary of polys that make up any airport video maps
+         * List of map layers that make up any airport video maps
          *
          * @property maps
-         * @type {object}
+         * @type {array}
```

Comment:
* this is correct, but incomplete. `array` -> `array<MapModel>`
* I'd encourage you to rename this property as well: `map` -> `mapModelList`

In the future, it may make sense to add an additional layer here known as a `Collection`.  So, in a possible future state, this would be instantiated as `this.maps = new MapLayerCollection(...args);`.  Probably isn't needed now, but its a strategy we've used elsewhere to encapsulate domain logic.

Reply:
I did wonder, but again say array being used elsewhere so assumed that was the convention. I did consider a `MapLayerCollection`, but wanted to get the core idea together first.

Reply:
In fact, the name layer is pretty much redundant. The model is already contained in an object called `maps`, so perhaps MapModel is better.

Reply:
I agree the layer is not needed in the name. `MapModel` and `.mapCollection`

Reply:
As suggested, I've added MapCollection too